reference-implementation/lib/transform-stream.js

```diff
@@ -363,14 +386,18 @@ module.exports = class TransformStream {
 
     this._readable = new ReadableStream(source, transformer.readableStrategy);
 
-    const sink = new TransformStreamSink(this, startPromise);
+    const sink = new TransformStreamSink(this, startPromise.then(() => {
+      return TransformStreamBackpressureGonePromise(this);
+    }));
```

Comment:
The change isn't fully reverted; the original explanatory comment is gone, and the code didn't need to be in a block per [ricea's review at the original PR](https://github.com/whatwg/streams/pull/550#discussion_r84678242).